XML3-Homework/hering_03.xml

```diff
@@ -0,0 +1,258 @@
+<root>
+    <section num="1" rhySchemes="11">
+        <line num="1"><num value="1">One</num> fish, <num value="2">Two</num> fish, <color>Red</color> fish, <color>Blue</color>
+            fish,</line>
+        <line num="2"><color>Black</color> fish, <color>Blue</color> fish, Old fish, New fish.</line>
+        <line num="3">This one has a little <rhy scheme="1">car</rhy>.</line>
+        <line num="4">This one has a little <rhy scheme="1">star</rhy>.</line>
+        <line num="5">Say! What a lot of fish there <rhy scheme="1">are</rhy>.</line>
+        <line num="6">Yes. Some are <color>red</color>, and some are <color><rhy scheme="2">blue</rhy></color>.</line>
+        <line num="7">Some are old and some are <rhy scheme="2">new</rhy>.</line>
+        <line num="8">Some are <rhy scheme="3">sad</rhy>, and some are <rhy scheme="3">glad</rhy>,</line>
+        <line num="9">And some are very, very <rhy scheme="3">bad</rhy>.</line>
+        <line num="10">Why are they <rhy scheme="3">sad</rhy> and <rhy scheme="3">glad</rhy> and <rhy scheme="3">bad</rhy>?</line>
+        <line num="11">I do not know, go ask your <rhy scheme="3">dad</rhy>.</line>
+        <line num="12">Some are thin, and some are <rhy scheme="4">fat</rhy>.</line>
+        <line num="13">The <rhy scheme="4">fat</rhy> one has a <color>yellow</color> <rhy scheme="4">hat</rhy>.</line>
+        <line num="14">From <rhy scheme="5">there</rhy> to here,</line>
+        <line num="15">From here to <rhy scheme="5">there</rhy>,</line>
+        <line num="16">Funny things are <rhy scheme="5">everywhere</rhy>.</line>
+        <line num="17">Here are some who like to <rhy scheme="6">run</rhy>.</line>
+        <line num="18">They run for fun in the hot, hot <rhy scheme="6">sun</rhy>.</line>
+        <line num="19">Oh me! Oh <rhy>my</rhy>! Oh me! oh <rhy scheme="7">my</rhy>!</line>
+        <line num="20">What a lot of funny things go <rhy scheme="7">by</rhy>.</line>
+        <line num="21">Some have <num value="2">two</num> feet and some have <num value="4"><rhy scheme="8">four</rhy></num>.</line>
+        <line num="22">Some have <num value="6">six</num> feet and some have <rhy scheme="8">more</rhy>.</line>
+        <line num="23">Where do they come from? I can't <rhy scheme="9">say</rhy>.</line>
+        <line num="24">But I bet they have come a long, long <rhy scheme="9">way</rhy>.</line>
+        <line num="25">we see them come, we see them <rhy scheme="10">go</rhy>.</line>
+        <line num="26">Some are fast. Some are <rhy scheme="10">slow</rhy>.</line>
+        <line num="27">Some are high. Some are <rhy scheme="10">low</rhy>.</line>
+        <line num="28">Not one of them is like <rhy scheme="11">another</rhy>.</line>
+        <line num="29">Don't ask us why, go ask your <rhy scheme="11">mother</rhy>.</line>
+    </section>
+<!--luh: I didn't have the time to add the scheme attributes to all the rhyme elemens. This book was longer than I thought-->
```

Comment:
@luh429 I can see this is some ambitious markup to follow a rhyme scheme, and I'm glad you left a comment indicating how you'd continue if you'd had time. The "proof of concept" is certainly fine for our homework, but it suggests an interesting possibility for a team project!